data/json/monsters/zed_misc.json

```diff
@@ -527,6 +527,130 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_nomex_1",
+    "type": "MONSTER",
+    "name": { "str": "nomex zombie" },
+    "description": "This was at some point a zombie wearing some kind of fire-resistant protective gear.  That same outfit is now shredded and has become part of its outer tissue, making it more resilient against physical attacks and heat-induced damage.  It also seems to be rather muscle packed.  At least its overgrown and hardened faceplate still holds up, preventing it from biting you and also rendering its eyes impaired.  However, it looks like it can still hear and smell you just fine.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "72500 ml",
+    "weight": "85500 g",
+    "hp": 120,
+    "speed": 70,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "yellow",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 3,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "armor_bash": 10,
+    "armor_cut": 10,
+    "armor_bullet": 8,
+    "armor_acid": 4,
+    "armor_fire": 15,
+    "armor_elec": 5,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_turnout_gear", "wps_humanoid_full_helmet_armored" ],
+    "families": [ "prof_intro_biology", "prof_wp_zombie", "prof_wp_syn_armored", "prof_wp_nat_armored" ],
+    "bleed_rate": 50,
+    "vision_day": 10,
+    "vision_night": 5,
+    "harvest": "zombie_nomex",
+    "special_attacks": [
+      {
+        "id": "slam",
+        "cooldown": 15,
+        "move_cost": 240,
+        "range": 1,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 12 } ],
+        "throw_strength": 10
+      },
+      {
+        "id": "headbash",
+        "cooldown": 20,
+        "move_cost": 300,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 16 } ]
+      }
+    ],
+    "upgrades": { "half_life": 42, "into": "mon_zombie_nomex_2" },
+    "death_drops": "mon_zombie_fireman_death_drops",
```

Comment:
This probably includes some turnout gear that this zombie doesn't have anymore since it's merged into its body